### PR TITLE
Add Steampunk Spotter to Other tools list

### PR DESCRIPTION
**Pull request by anzoman**
##### SUMMARY
Add [Steampunk Spotter](https://spotter.steampunk.si/) tool that analyzes and offers recommendations for Ansible Playbooks.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: docs/docsite/rst/community/other_tools_and_programs.rst [hunk 1]
```diff
@@ -120,5 +120,6 @@ Other tools
 - `Mitogen for Ansible <https://mitogen.networkgenomics.com/ansible_detailed.html>`_ - uses the `Mitogen <https://github.com/dw/mitogen/>`_ library to execute Ansible playbooks in a more efficient way (decreases the execution time).
 - `nanvault <https://github.com/marcobellaccini/nanvault>`_ - a standalone tool to encrypt and decrypt files in the Ansible Vault format, featuring UNIX-style composability.
 - `OpsTools-ansible <https://github.com/centos-opstools/opstools-ansible>`_ - uses Ansible to configure an environment that provides the support of `OpsTools <https://wiki.centos.org/SpecialInterestGroup/OpsTools>`_, namely centralized logging and analysis, availability monitoring, and performance monitoring.
+- `Steampunk Spotter <https://steampunk.si/spotter>`_ - provides an Assisted Automation Writing tool that analyzes and offers recommendations for your Ansible Playbooks.
```
Comment (samccann) [inline docs/docsite/rst/community/other_tools_and_programs.rst]:
```suggestion
- `Steampunk Spotter <https://pypi.org/project/steampunk-spotter/>`_ - provides an Assisted Automation Writing tool that analyzes and offers recommendations for your Ansible Playbooks.
```

Comment (samccann) [reply docs/docsite/rst/community/other_tools_and_programs.rst]: Preference is for the open source version. Thanks for adding this!

Comment (anzoman) [reply docs/docsite/rst/community/other_tools_and_programs.rst]: @samccann thanks, I have incorporated your suggestion.

**Review by oraNod — APPROVED** [review]
LGTM

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @anzoman for the Ansible docs fix and welcome to the Ansible project!

**Pull request merged**